src/utils/processOpenClientResponse.ts

```diff
@@ -0,0 +1,18 @@
+import { EventResponse } from '@fingerprintjs/fingerprintjs-pro-server-api'
+import { plugins, PluginType } from './registerPlugin'
+
+export type ProcessOpenClientResponseContext = {
+  event: EventResponse | null
+  sealedResult: string
+}
+
+export async function processOpenClientResponse(context: ProcessOpenClientResponseContext): Promise<void> {
+  const filteredPlugins = plugins.filter((t) => t.type === PluginType.ProcessOpenClientResponse)
+  for (const filteredPlugin of filteredPlugins) {
+    try {
+      await filteredPlugin.function(context)
+    } catch (e: any) {
```

Comment:
nitpick: let's not use `any`. If we are unsure, let's use `unknown`

Reply:
Refactored by this commit: https://github.com/fingerprintjs/fingerprint-pro-fastly-proxy-integration-example/pull/5/commits/153852815fbf6d80aed5a3c0d7550ef95e5bbb79